pkgs/by-name/ba/bashrun2/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  makeWrapper,
+  xorg,
+  ncurses,
+  coreutils,
+  bashInteractive,
+  gnused,
+  gnugrep,
+  glibc,
+  xterm,
+  util-linux,
+}:
+
+stdenv.mkDerivation {
+  pname = "bashrun2";
+  version = "0.2.6-unstable-2024-12-09";
+
+  src = fetchFromGitHub {
+    owner = "Dopplerian"; # Using my repo since I did a small fix for this to work in NixOS. If it gets merged, this can be changed to hbekel's original repo.
```

Comment:
You should still fetch from the original repo (ideally using a tag), and apply the patches needed for NixOS similar to what you do below. If they are significant you can store in a file in the folder

There is an example here - https://github.com/NixOS/nixpkgs/blob/nixos-24.11/pkgs/development/compilers/go/1.22.nix#L71